.github/workflows/release.yaml

```diff
@@ -0,0 +1,42 @@
+name: "Release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseBranch:
```

Comment:
I think this could be inferred from the new version (not a blocker)

Reply:
yes I know, it could be way smarter, technically you don't need anything, you can simply increment minor. I just wanted to check it works with `workflow_dispatch`

Reply:
Done